lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -477,12 +494,17 @@ void SVExtractTestCodeImplPass::runOnOperation() {
         continue;
       }
 
-      doModule(rtlmod, isAssert, "_assert", assertDir, assertBindFile,
-               testBenchDir, instanceGraph);
-      doModule(rtlmod, isAssume, "_assume", assumeDir, assumeBindFile,
-               testBenchDir, instanceGraph);
-      doModule(rtlmod, isCover, "_cover", coverDir, coverBindFile, testBenchDir,
-               instanceGraph);
+      bool anyThingExtracted = false;
+      anyThingExtracted |= doModule(rtlmod, isAssert, "_assert", assertDir,
+                                    assertBindFile, bindTable);
+      anyThingExtracted |= doModule(rtlmod, isAssume, "_assume", assumeDir,
+                                    assumeBindFile, bindTable);
+      anyThingExtracted |= doModule(rtlmod, isCover, "_cover", coverDir,
+                                    coverBindFile, bindTable);
```

Comment:
Mega nit: if slightly shorter variables names are used this can likely get to:

```c++
bool anythingExtracted = 
  doModule(...) |
  doModule(...) |
  doModule(...);
```

Reply:
bitwise-or seems not to guarantee the evaluation order so I think it would be safer to keep as is since doModule mutates IR.

Reply:
It's not pretty, but I think the point about evaluation order makes sense, so I'll keep this as-is.